packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -204,7 +230,7 @@ Include this section if there are known issues that apply to the entire configur
 
 You can't publish integration with the following Okta features in the OIN catalog:
 
-* **SWA apps:** Okta no longer publishes new Secure Web Authentication (SWA) integrations to the OIN catalog. Existing SWA integrations are still maintained by the OIN team. To update an existing SWA integration, contact the [OIN team](mailto:oin@okta.com).
+* **SWA apps:** Okta no longer publishes new Secure Web Authentication (SWA) integrations to the OIN catalog. The OIN team maintains existing SWA integrations. To update an existing SWA integration, contact the [OIN team](mailto:oin@okta.com).
```

Comment:
Can we remove `To update an existing SWA integration, contact the [OIN team](mailto:oin@okta.com).` ?

Since this is an ISV guide, I don't think they need to know how to report broken SWA, that is an instruction for WIC end users who are consuming SWA integrations.

Reply:
Yes, I can remove this.
I thought this line was for ISVs who already have existing SWA integrations and want to make an edit in the OIN Manager? I vaguely remember that the DAX team asked us to add this line a few months ago.